sslcontext-kickstart/src/main/java/nl/altindag/ssl/util/KeyStoreUtils.java

```diff
@@ -56,6 +60,14 @@ public final class KeyStoreUtils {
     private static final UnaryOperator<String> KEYSTORE_NOT_FOUND_EXCEPTION_MESSAGE = certificatePath -> String.format("Failed to load the keystore from the classpath for the given path: [%s]", certificatePath);
     private static final String EMPTY_TRUST_MANAGER_FOR_TRUSTSTORE_EXCEPTION = "Could not create TrustStore because the provided TrustManager does not contain any trusted certificates";
     private static final String EMPTY_CERTIFICATES_EXCEPTION = "Could not create TrustStore because certificate is absent";
+    private static final List<Path> LINUX_CERTIFICATE_PATHS = Stream.of(
+                    "/etc/ssl/certs/ca-certificates.crt",
+                    "/etc/ssl/certs",
+                    "/usr/local/share/ca-certificates",
+                    "/usr/share/ca-certificates",
+                    "/etc/pki/ca-trust/source/anchors/")
```

Comment:
Should we annotate which one is considered under which distro? Nice gathering!!

Reply:
Yes, that would be really helpful as it will sort of be a documentation for it. I will do that! Thank you